subsys/lorawan/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Kconfig - LoRaWAN configuration options
+#
+# Copyright (c) 2018 Ding Tao
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menu "LoRaWAN"
+
+config LORAWAN
```

Comment:
Merge this line with line 8: 

menuconfig LORAWAN
 ....

Reply:
Write this as you suggested, this menu will not be shown in a single page.